app/src/ui/add-repository/create-repository.tsx

```diff
@@ -91,7 +91,7 @@ interface ICreateRepositoryState {
 export class CreateRepository extends React.Component<
   ICreateRepositoryProps,
   ICreateRepositoryState
-> {
+  > {
```

Comment:
@noelledusahel thanks for starting this off! It looks like there's some formatting changes that have been introduced by the editor. To tidy these up, you can run `yarn lint:fix` locally and commit the changes.

Reply:
will do, thanks!